api2-war/src/main/java/eu/europeana/api2/v2/model/translate/TranslationMap.java

```diff
@@ -0,0 +1,48 @@
+package eu.europeana.api2.v2.model.translate;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.validation.constraints.NotNull;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class TranslationMap extends LinkedHashMap<String, List<String>> {
+
```

Comment:
same a above so far only SR API is using it. It's not used in Translation API. But we can do this in the future when we have other api's using this class. There are chances that it might be different for other APIs